### ISSUE TITLE
Delete Jekyll/Ruby

### ISSUE THREAD

**Issue by aravindet** [speakforme/frontend#138]
This should make it easier for new contributors to get started, and also make deploys faster.

**Comment by arincode** [comment]
I agree. Plus a readme will help a lot for new less experienced guys like me

**Comment by jace** [comment]
@arincode This repo is currently unmaintained as the SpeakForMe project has disbanded. We learnt that petitioning Parliament is ineffective in the current political climate.
